### DISCUSSION TITLE
Import buttons missing from Firefly III Data Import Tool page

### DISCUSSION THREAD

**Discussion by skanur** [firefly-iii/firefly-iii#5387] [Firefly III] [answered]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

I don't see import buttons for importing CSV files or importing from Nordigen or Spectre.

### Expected behaviour

Here's the image of how the webpage looks like on my browser.

![image](https://user-images.githubusercontent.com/5601252/145687422-f0f0bcfd-b5ca-4657-91c6-76db111c8386.png)

### Debug information

Debug information generated at 2021-12-11 19:26:15 Europe/Amsterdam for Firefly III Data Importer version **0.3.0**.

| Scope | Version |
| --- | --- |
| FIDI | 0.3.0 |
| PHP | 8.0.12 |
| Host | Linux |

| System info | Value |
| --- | --- |
| Using docker? | true |
| System TZ |  |
| Browser TZ | Europe/Helsinki |
| App environment | production |
| App debug mode | false |
| App cache driver | file |
| App logging | debug, stack |
| Display errors | Off |
| Error reporting | ALL errors |
| Interface | apache2handler |
| BCscale | 12 |
| Trusted proxies (.env) |  |

| User info | Value |
| --- | --- |
| User agent | Mozilla/5.0 (X11; Linux x86_64; rv:95.0) Gecko/20100101 Firefox/95.0 |

### Additional info

I use Firefox 95.0 on Linux platform. I have also tried Chromium on same platform with same response.
I have also tried Microsoft Edge and Internet explorer on Windows 11 machine with same experience.

**Comment by JC5** [comment, marked as answer]
If you view the source of the page, you may see the links to the CSS files. Are they correct?

**Reply by skanur** [reply]
These are the two links to .css files I see in the web page:
```
<!-- Bootstrap CSS -->
    <link rel="stylesheet" href="http://issue5387.skanur.me/css/bootstrap.min.css?version=0.3.0">
    <link href="http://issue5387.skanur.me/fa/css/all.min.css" rel="stylesheet">

    <title>Firefly III Data Importer // Index</title>
</head>
```

And both of the link are valid.

My web-dev skills are non-existent, not sure how much I can help you. I have temporarily exposed the instance at https://issue5387.skanur.me. I will take it down at later point (please advice me if this is a bad idea)

**Reply by JC5** [reply]
Your reverse proxy is programmed incorrectly, the links are http but you point me to a https website.

https://docs.firefly-iii.org/data-importer/install/self_hosted/#reverse-proxies

**Reply by skanur** [reply]
Thanks. That was indeed the problem. Page loads normally now!
